android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -290,6 +299,58 @@ public void captureEnvelope(ReadableArray rawBytes, ReadableMap options, Promise
         promise.resolve(true);
     }
 
+    @ReactMethod
+    public void captureScreenshot(Promise promise) {
+        final Activity activity = this.getReactApplicationContext().getCurrentActivity();
```

Comment:
Maybe we could refactor `ScreenshotEventProcessor` on android that takes an `Activity` as a parma and reuse this class/method in here, so we avoid duplication and increase of bundle size.